core/conn/trafci/installer_pom.xml

```diff
@@ -1,106 +1,112 @@
-<!-- 
- @@@ START COPYRIGHT @@@                                                       
-
- Licensed to the Apache Software Foundation (ASF) under one
- or more contributor license agreements.  See the NOTICE file
- distributed with this work for additional information
- regarding copyright ownership.  The ASF licenses this file
- to you under the Apache License, Version 2.0 (the
- "License"); you may not use this file except in compliance
- with the License.  You may obtain a copy of the License at
-
-   http://www.apache.org/licenses/LICENSE-2.0
-
- Unless required by applicable law or agreed to in writing,
- software distributed under the License is distributed on an
- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- KIND, either express or implied.  See the License for the
- specific language governing permissions and limitations
- under the License.
-
- @@@ END COPYRIGHT @@@
--->
+<!-- @@@ START COPYRIGHT @@@ Licensed to the Apache Software Foundation (ASF) 
```

Comment:
Not sure what happened to the license text here. Could we put the original back?